plugins/python-build/bin/python-build

```diff
@@ -1470,7 +1470,9 @@ build_package_symlink_version_suffix() {
   if [[ "$PYTHON_CONFIGURE_OPTS" == *"--enable-framework"* ]]; then
     if [ -e "${PREFIX_PATH}/bin" ]; then
       # Always create `bin` as symlink to framework path if the version was built with `--enable-frameowrk` (#590)
-      mv -f "${PREFIX_PATH}/bin" "${PREFIX_PATH}/bin.orig"
+      if [ ! -e "${PREFIX_PATH}/bin.orig" ]; then
```

Comment:
I overlooked that there `[ -e "${PREFIX_PATH}/bin" ]` exists at L1471.... Obviously we don't need to duplicate the line.